incs/parser.h

```diff
@@ -162,7 +164,7 @@ static const t_token g_tokens[] = {
 	{">",	1,	TT_REDIR,		TC_CHEV_RIGHT,		token_parse_chev_right},
 	{"<",	1,	TT_REDIR,		TC_CHEV_LEFT,		token_parse_chev_left},
 	{"|",	1,	TT_REDIR,		TC_PIPE,			token_parse_pipe},
-	{"&",	1,	TT_SPECIAL,		TC_AND,				token_parse_none},
+	{"&",	1,	TT_SPECIAL,		TC_AND,				token_parse_and},
```

Comment:
Need to parse the token & for support of 'ls &>out'.
If not the & is skipped (such as a token_none).

See token_parse_and